manager/apis/app/v1beta1/blueprint_types.go

```diff
@@ -47,6 +47,29 @@ type ModuleArguments struct {
 	Assets []AssetContext `json:"assets,omitempty"`
 }
 
+// ModuleDeployment specifies the deployment details of the module
+type ModuleDeployment struct {
+	// Cluster name
+	Cluster string `json:"cluster"`
+	// Release name
+	Release string `json:"release"`
+	// URLs the module needs access to
+	URLs []string `json:"urls"`
```

Comment:
what will be format of the URLs? Is it net.URL,  just hostname, or any format defined in the taxonomy?

Reply:
For service it will probably be release.namespace:port. For datasets, either domain or host:port. Depends on whatever details we get from the catalog...